raft/server.cc

```diff
@@ -461,7 +461,7 @@ future<> server_impl::wait_for_state_change(seastar::abort_source* as) {
     try {
         return as ? _state_change_promise->get_shared_future(*as) : _state_change_promise->get_shared_future();
     } catch (abort_requested_exception&) {
-        throw request_aborted();
+        throw request_aborted(format("Aborted while waiting for state change on server: {}, latest applied entry: {}", _id, _applied_idx));
```

Comment:
Would it be hard to print the current state from `raft::fsm`?

Reply:
Done, added a function for that!